WordPress/Classes/ViewRelated/People/PeopleViewController.swift

```diff
@@ -293,24 +293,31 @@ open class PeopleViewController: UITableViewController, NSFetchedResultsControll
         }
 
         var result: (retrieved: Int, shouldLoadMore: Bool)?
+        var loadError: Error?
 
         let group = DispatchGroup()
         group.enter()
         peopleService.loadUsersPage(offset, success: { (retrieved, shouldLoadMore) in
             result = (retrieved, shouldLoadMore)
             group.leave()
-        }, failure: { (error) in
+        }, failure: { error in
+            loadError = error
             group.leave()
         })
 
         group.enter()
         roleService.fetchRoles(success: {_ in
             group.leave()
-        }, failure: { _ in
+        }, failure: { error in
+            loadError = error
             group.leave()
         })
 
-        group.notify(queue: DispatchQueue.main) {
+        group.notify(queue: DispatchQueue.main) { [weak self] in
+            if let error = loadError {
+                self?.presentAlert(error)
```

Comment:
I would rather we don't display an alert but use the `NoResultsView` to display the fact that there's a network connectivity problem, what do you think?

Reply:
Alright, so all alerts go away, and we'll only use `NoResultsView`. Go it.